lib/landable/engine.rb

```diff
@@ -36,15 +36,15 @@ class Engine < ::Rails::Engine
       end
     end
 
-    initializer "landable.seed_required" do |app|
+    initializer "landable.seed_required" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.